File: 프로그래머스/lv1/131705. 삼총사/삼총사.py
def solution(number):
    n = len(number)
    answer = 0
    
    for i in range(n):
        for j in range(i + 1, n):
            for k in range(j + 1, n):
                if number[i] + number[j] + number[k] == 0: answer += 1
    
    return answer

File: 프로그래머스/lv1/131705. 삼총사/삼총사.js
function solution(number) {
    const n = number.length
    let answer = 0
    
    // 완전탐색
    for (let i = 0; i < n; i++) {
        for (let j = i + 1; j < n; j++) {
            for (let k = j + 1; k < n; k++) {
                if (number[i] + number[j] + number[k] === 0) {
                    answer += 1
                }
            }
        }
    }
    
    return answer
}